zero/ifc/text/format.cppm

```diff
@@ -0,0 +1,88 @@
+/**
+* Provides functions and enums for string formatting and styling
+*/
+
+export module format;
+
+import std;
+
+template<typename T>
+std::string stringify(const T& t) {
+    std::ostringstream oss;
+    oss << t;
+    return oss.str();
+}
+
+export namespace zero::fmt {
+
+    enum class Color {
```

Comment:
It w'd be fine to have the enumerations in another module. Since the `stylize` funcion is related, I think that we can group them in a new module, both the function and the enumerators for colors/styles